lib/kitchen/provisioner/chef_base.rb

```diff
@@ -554,8 +555,14 @@ def omnibus_dir_option
       end
 
       def install_from_file(command)
```

Comment:
This change looks right in context, but do you know why the original code doesn't directly execute the mixlib install script when it's available? 

It seems odd that we're executing a script that writes a script and executes it, instead of either - 1) uploading the script to the root path and executing it by name as the install_command, or 2) execute the install script itself directly, instead of writing to file at all.

Reply:
TK was originally written before mixlib-install existed - thats my only guess for why this is the way it is